src/components/ViewerToolbar.jsx

```diff
@@ -16,7 +17,14 @@ export const ViewerToolbar = ({ t, router, hidden }) => {
   }
   return (
     <div className={classNames(styles['pho-viewer-toolbar'], {[styles['--hidden']]: hidden})} role='viewer-toolbar'>
-      <div className={styles['pho-viewer-toolbar-actions']} />
+      <div className={classNames(styles['coz-selectionbar'], styles['pho-viewer-toolbar-actions'])}>
+        <button
+          className={styles['coz-action-download']}
+          onClick={() => downloadFile(currentPhoto)}
```

Comment:
I tried to be the more explicit I can for this one, I will speak about coupling: `downloadFile` comes from outside of this function scope (the functional component): imports are in the "file" scope, functional component create another scope.

If functions use variable from outside its scope, it is not a pure function. And if the function is not pure, then it is sensitive to side effects.

To avoid this, we should give `downloadFile` to the component through its props. And to give props to a react component, there is two ways:
- passing it through its parent like that: `<ViewerToolbar downloadFile={downloadFile} />`
- passing it through the react-redux connect HOC like that: `connect(null, { downloadFile })(ViewerToolbar)`

As we use a redux architecture, we may prefer the second solution. What do you think?